app/scripts/views/complete_sign_up.js

```diff
@@ -94,12 +96,14 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
               return false;
             }
 
-            return self.getAccount().isSignedIn()
+            var account = self.getAccount();
+            return account.isSignedIn()
               .then(function (isSignedIn) {
                 if (isSignedIn) {
                   self.navigate('settings', {
                     success: t('Account verified successfully')
                   });
+                  self._notifications.broadcast(self._notifications.EVENTS.SIGNED_IN, account.toJSON());
```

Comment:
I'm still asking myself whether all brokers/reliers should call `isSignedIn` and broadcast this message if so. I have this suspicion that we could miss some edge cases because of the `if (! self.relier.isDirectAccess()) {` branch on line 94.

Leave this for now, we can revisit if needed.